spec/src/main/asciidoc/architecture.asciidoc

```diff
@@ -33,12 +33,33 @@ In reverse chronological order, here's the specification for each MicroProfile r
 [[microprofile2.0.1]]
 === MicroProfile 2.0.1 (3Q2018)
 
-MicroProfile 2.0.1 [1] is the sixth release for the https://projects.eclipse.org/projects/technology.microprofile[Eclipse MicroProfile project].
+MicroProfile 2.0.1 is the latest release for the https://projects.eclipse.org/projects/technology.microprofile[Eclipse MicroProfile project].
```

Comment:
If you say latest release, we need to keep on changing it when we do MP 2.1 release. How about just deleting 'latest'?

Reply:
Good point, @Emily-Jiang!  I will revert "latest" to numbering the releases like we had done in the past.  Thanks.